Sources/OpenTelemetrySdk/Trace/SpanBuilderSdk.swift

```diff
@@ -33,7 +33,7 @@ class SpanBuilderSdk: SpanBuilder {
 
     private var startAsActive: Bool = false
 
-    private var startTime = Date()
```

Comment:
where is this initialization happening now? This PR only removes initialization, not adding another one instead?

Reply:
It looks like the constructor of `RecordEventsReadableSpan` had already taken an optional `startTime` in construction, but this behavior wasn't propagated. By making it optional in the `SpanBuilderSdk` it now is behaving as expected.  https://github.com/open-telemetry/opentelemetry-swift/pull/282/files#diff-2b0cc00ea3cbfbf944d9c31aa9fe1760f6414f213bc8bcc64a5850d6652a366fL112

Reply:
Optional types (in this case the Date? is an optional Date type) are always nil if not initialised. The value will only be set if the user explicitly calls setStartTime(). If The builder starts the span with the startTime uninitialised, the span will use the Clock assigned to it instead of this value

Reply:
Line 112 of `RecordEventsReadableSpan`

Reply:
thank you for clarification!

Reply:
Yes, this value should only be used if the user explicitly wants to fix the start time, by default the span should use the time that the assigned Clock has